cpp/test/sg/rf_batched_regression_test.cu

```diff
@@ -136,7 +136,7 @@ const std::vector<RfInputs> inputs = {
   {1000, 10, 10, 1.0f, 1.0f, 12, -1, true, false, 10,
    SPLIT_ALGO::GLOBAL_QUANTILE, 2, 2, 0.0, 2, CRITERION::MAE, 0.7f},
   {2000, 20, 20, 1.0f, 0.6f, 13, -1, true, false, 10,
-   SPLIT_ALGO::GLOBAL_QUANTILE, 2, 2, 0.0, 2, CRITERION::MSE, 0.7f}};
+   SPLIT_ALGO::GLOBAL_QUANTILE, 2, 2, 0.0, 2, CRITERION::MSE, 0.68f}};
```

Comment:
Note to the reviewer: I am lowering the accuracy requirement to 0.68, as I've introduced additional regularization that caused a slight accuracy drop for this example.